lib/gc/Transforms/GPU/LinalgToXeGPU.cpp

```diff
@@ -1405,6 +1405,17 @@ LogicalResult createMemoryFillKernel(linalg::LinalgOp linalgOp,
   auto outputType = cast<ShapedType>(output.getType());
   auto outputShape = outputType.getShape();
 
+  if (outputShape.size() != 2) {
+    return rewriter.notifyMatchFailure(
+        linalgOp, "Memory fill operation expects 2D output");
+  }
+
+  // Otherwise 'xegpu-to-vc' pass will fail to convert it to VC
+  if (outputShape[0] * outputShape[1] < 16) {
```

Comment:
[`gpu-runner/mlp.mlir`](https://github.com/intel/graph-compiler/blob/main/test/mlir/test/gc/gpu-runner/mlp.mlir) test contains a [`linalg.fill() tensor<1x1xf32>`](https://github.com/intel/graph-compiler/blob/be1bcb92a087cd12bd3079fde6220d1efaed7e6a/test/mlir/test/gc/gpu-runner/mlp.mlir#L85) which is to small to be processed via xegpu